README.md

```diff
@@ -72,4 +72,13 @@ convivaConfig.setCustomData(customMapOfKeyValuePairs);
 
 ```
 
+#### Background handling
+
+If your app stops playback when entering background conviva suggests to end the active session. Since the integration can't know if you app supports playback in background this can't be done automatically.
```

Comment:
```suggestion
If your app stops playback when entering background conviva suggests to end the active session. Since the integration can't know if your app supports background playback this can't be done automatically.
```